lib/Configuration.py

```diff
@@ -1100,9 +1100,35 @@ def FindLatestManifest(self, train=None, require_signature=False):
             else:
                 train = temp_mani.Train()
 
+        if enterprise:
+            mani_file = self.TryGetNetworkFile(url="%s/%s/LATEST-ENTERPRISE" % (self.UpdateServerMaster(), train),
+                                               reason="GetLatestManifest")
+            if mani_file is None:
+                log.debug("Could not get latest enterprise manifest file for train %s" % train)
+                return None
+
+            rv = Manifest.Manifest(self, require_signature=require_signature)
+            rv.LoadFile(mani_file)
+            mani_file.close()
+
+            update_rate = rv.dict().get("rate")
+            if update_rate is not None:
+                if os.path.exists("/data/rate"):
```

Comment:
Not sure what is the intent of have the rate set locally?

Reply:
Debugging, testing

Reply:
That would be just a flag, not a percent?

Reply:
Done